### PR TITLE
Font Updates

### PR DESCRIPTION
**Pull request by rjspiker**
Preview: https://packer-fonts.netlify.com

This PR accomplishes the following:
- Add newly approved fonts (Gilmer, Metro Sans, Deja Vu) and remove deprecated fonts
- Global typography updates (font-size, line-height, etc), adding `.g-type` classes shared across sites
- Color updates to match color scheme with other Hashicorp sites
- Button style overhaul

### CODE REVIEW

File: website/source/assets/stylesheets/_fonts.scss [hunk 1]
```diff
@@ -0,0 +1,83 @@
+/* Web Fonts */
+@font-face {
+  font-family: "klavika-web";
+  src: url("/assets/fonts/klavika/500.woff2") format("woff2"),
+    url("/assets/fonts/klavika/500.woff") format("woff");
+  font-weight: 700;
+  font-style: normal;
+}
```
Comment (pruett) [inline website/source/assets/stylesheets/_fonts.scss]: Question. Will we still need Klavika going forward?

Comment (rjspiker) [reply website/source/assets/stylesheets/_fonts.scss]: For Vault I think we do, but for the other .io sites we shouldn't.  Thanks for calling that out @pruett 🙌.  I'll get that removed.

**Comment by kylemac** [comment]
LGTM w/ exception of my more general `g-type-display-3` feedback from other PRs.

**Review by zchsh — APPROVED** [review]
✅ Deploy preview looks great.

I found a typo that originated on the design side I think - `g-type-buttons-and-standlone-links`, where `standlone` should probably be `standalone`. Just flagging, I definitely think this is something we can leave for now if it's a hassle to change it. Or if it's an easy find & replace, that might be worth it to avoid future confusion (and apologies for passing the stylesheet over with this typo, and for not catching it earlier!)

**Comment by rjspiker** [comment]
@zchsh ha ya, it looks like it's across the board.  I definitely would hate to leave it there.  I'll do a find/replace so we can 🔪 it.  Thanks.

**Comment by rjspiker** [comment]
Thanks for the feedback on this.  Going to open a PR upstream.

**Pull request merged**